### ISSUE TITLE
users_uin_institution_id_key errors on UID/UIN changes

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#6992]
I've observed that students who have had a changed UID (but a consistent UIN) can give these `users_uin_institution_id_key` errors on login. I thought we'd fixed this to automatically update in the past, but it seems to still be a problem.

Currently I talk to @mwest1066 to get these fixed, but it's a support burden. Can we make PL do the right thing in this case? Or at least automate the reporting/fixing of these issues?

**Comment by trombonekenny** [comment]
In person, @mwest1066 explained this is likely tied to PrairieTest and the account creation scenarios that exist from that usage, which is why it's showing up as a "new" issue again.

**Comment by nwalters512** [comment]
For folks with access to the PrairieTest repo, we're tracking this there as well: https://github.com/PrairieLearnInc/PrairieTest/issues/403

**Comment by trombonekenny** [comment]
@mwest1066 This is resolved now, right, through the https://github.com/PrairieLearnInc/PrairieTest/issues/403 issue being closed and the recent fixes/testing fixes with `users_select_or_insert`?

**Comment by reteps** [comment]
Is this resolved?

**Comment by mwest1066** [comment]
Yes, this was fixed by https://github.com/PrairieLearnInc/PrairieTest/pull/898
